src/app/content/components/BookBanner.spec.tsx

```diff
@@ -26,7 +26,7 @@ describe('BookBanner', () => {
     resetModules();
 
     window = assertWindow();
-    delete window.location;
+    delete (window as any).location;
```

Comment:
why are all these as any casts here?

Reply:
it was added due to the some rule from typescript 4^ that disallows removing properties that are not marked as optional
https://github.com/openstax/rex-web/pull/1027#pullrequestreview-560422300

Reply:
So they aren't necessary for the 3.9 upgrade?

Reply:
I think they are not required for 3.9 but from what I've talked with Karina we will still want to update to ts ^4 after we update the `node`
Do you prefer to remove all unnecessary fixes for the current version?